### PR TITLE
Setup the project for react

### PR DESCRIPTION
**Pull request by ibugithub**
-In  this branch, We setup the project for react.

### CODE REVIEW

**Review by joel-kalema — COMMENTED** [review]
Hi @

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request  🛑 

pirates-of-the-caribbean-johnny-depp


- [ ] You have submitted a project that doesn't have professional README


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @joel-kalema in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,70 +1,142 @@
...
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
 
-### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+This project is [MIT](./LICENSE.md) licensed.
```
Comment (mikethreels) [inline README.md:139]: - [ ] the link to the MIT license is not working as expected, please make sure it is working similarly to the template provided by Microverse.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,142 @@
...
+To run the project, execute the following command:
+`npm start`
 
-### Code Splitting
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+### Run tests
+To run tests, run the following command:
+<p> No test available </p>
+<p>But you can inspect the project's webpages by the browser's developer mode </p>
```
Comment (mikethreels) [inline README.md]: - [ ] please make sure to update the readme file with information specific to this project.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Ibu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job setting up the project.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1,70 +1,142 @@
...
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## 🚀 Live Demo <a name="live-demo"></a>
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+- [Live Demo Link](https://ibugithub.github.io/math-Magician/)
```
Comment (wayungi) [inline README.md:55]: - [OPTIONAL] Your live demo link is broken. The link should lead to a hosted version of your project.  If project deployment is not a requirement, you can delete the **live demo** section altogether. If you however want to include the live demo, please deploy the project and add a link that links to the live version 👏🏾

**Review by wayungi — APPROVED** [review]
Hi @ibugithub , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] All linter checks are passing ✔️ 
- [ ]  Correct GitHub flow ✔️
- [ ] Good commit messages ✔️  
- [ ] Project up and running without errros ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**